### PR TITLE
Migrate dev.to article to this site: Certifications I'm considering, as an experienced software engineer

### LINKED ISSUE #72: Migrate dev.to article to this site: Certifications I'm considering, as an experienced software engineer
Title: Certifications I'm considering, as an experienced software engineer

Part of #55. Reference PRs and Issues on how to repeat this, basically: https://github.com/ScalaWilliam/ScalaWilliam.com/issues/61 https://github.com/ScalaWilliam/ScalaWilliam.com/issues/65 https://github.com/ScalaWilliam/ScalaWilliam.com/pull/67 https://github.com/ScalaWilliam/ScalaWilliam.com/pull/62

I'll pay $15 for this. Much simpler than Medium ones. It's posted on: http://work.scalawilliam.com/migrate-certifications-article/

Make sure to post "**I'm taking on this**" before starting to lock the piece of work to yourself.
Rules: http://work.scalawilliam.com/rules/

### PR DESCRIPTION
**Pull request by Zegnat**
Closes #72.

Nothing spectacular here.

* Added `target="_blank"` to all links outside of ScalaWilliam.com, as that seems to be the norm.
* Wrote a bare minimal `description` in the meta data.

### CODE REVIEW

**Comment by ScalaWilliam** [comment]
Merge conflict now.

**Comment by Zegnat** [comment]
Rebased on new h-feed.

Hopefully git understands that these article migrations do not conflict with each other.

**Comment by ScalaWilliam** [comment]
Almost there :)

**Comment by Zegnat** [comment]
How about now…

File: 1704/certifications-im-considering/index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta content="text/html; charset=UTF-8" http-equiv="content-type">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700,900%7CRoboto+Mono" rel="stylesheet">
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/themes/prism.min.css">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.5.13/clipboard.min.js"></script>
```
Comment (ScalaWilliam) [inline 1704/certifications-im-considering/index.html]: Oh I just noticed some of these scripts might not be necessary as no code highlighting.

Comment (Zegnat) [inline 1704/certifications-im-considering/index.html]: Oh, that could be, I didn’t really check where you were using the separate JavaScript and CSS calls for. Just went with whatever was there in the base template you linked as example.

Comment (ScalaWilliam) [inline 1704/certifications-im-considering/index.html]: Sure. I'll merge, this can be a separate optimisation task later on anyway - across the board IMO.

**Pull request merged**